frontend/src/components/Content.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types'
+import { createUseStyles } from 'react-jss'
+import Theme from './Theme'
+
+const jss = createUseStyles( {
```

Comment:
```suggestion
const useStyles = createUseStyles( {
```

This is more explicit that `createUseStyles` returns a hook, and conventional usage of `react-jss`